.travis.yml

```diff
@@ -0,0 +1,20 @@
+language: rust
+
+sudo: false
+os:
+  - linux
+  - osx
```

Comment:
Oh, it also [fails on Linux](https://travis-ci.org/google/findutils/builds/209000807)...